### DISCUSSION TITLE
Pass context or customer details

### DISCUSSION THREAD

**Discussion by dev-orion-ps** [voxos-ai/bolna#228] [Q&A] [answered]
How customer information like Name, email, Address or Customer order history can be passed to call.

**Comment by prateeksachan** [comment, marked as answer]
What you can do here is store the context in some persistent database and retrieve it back in the websocket connection once the call connects.

You can pass it while initiating the `/call` assuming the said variables are defined within curly bracets `{}` in the prompt and store it.
```
"some_context_data": {
    "name": "XYZ",
    "email": "abc@abc.com"
}
```

Once the websocket connection happens, you can retrieve this payload and pass it in the `AssistantManager` instance.
https://github.com/bolna-ai/bolna/blob/9842c50241c5bc0be5a5f9fb4bf43f49b365f1fe/local_setup/quickstart_server.py#L86-L88

It will then automatically inject and set the context_data.
https://github.com/bolna-ai/bolna/blob/9842c50241c5bc0be5a5f9fb4bf43f49b365f1fe/bolna/agent_manager/assistant_manager.py#L11-L20

**Reply by dev-orion-ps** [reply]
Thanks @prateeksachan !
